src/makeRobot.js

```diff
@@ -39,6 +39,79 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    robotName: name,
+    robotWheels: wheels,
+    robotVersion: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.robotName}, `
+      + `chip version: ${this.robotVersion}, wheels: ${this.robotWheels}`;
+    },
+
+    get location() {
+      return `${this.robotName}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value) {
+      if (value <= 0) {
+        this.coords.y += 0;
+      } else if (value > 0) {
+        this.coords.y += value;
+      } else {
+        this.coords.y++;
+      }
```

Comment:
if you already check that value <= 0, you know that you have value > 0
so, you don't need to check it

```suggestion
      if (value > 0) {
        this.coords.y += value;
      }
```